pkg/trace/operation.go

```diff
@@ -129,6 +129,10 @@ func (o *Operation) ID() string {
 	return o.id
 }
 
+func (o *Operation) Auditf(format string, args ...interface{}) {
```

Comment:
Is there a design available for this work? I assume there's a reason to introduce `Auditf` as a distinct method (instead of just using `Infof`), but it's not clear from this PR.

Understanding the long-term plan will help with review of this PR.

Some initial thoughts:
* Maybe this method should call `Infof` to avoid duplicate code (or maybe that's a silly suggestion given the long-term plan)
* Maybe a `trace.BeginAndAudit` method (or just `trace.Audit`?) could be introduced that calls `trace.Begin` and `op.Auditf` to avoid the need to for two lines duplicating the operation name at each of these call sites

Reply:
Since these changes are annoying and mechanical, here's a PR: lcastellano/vic#2

Reply:
Just aisle conversations so far. The idea is to have an Audit method that records the incoming API request and in the future could record other interesting data (e.g. source IP address, ..). This logging operation would be reserved for API entry points and for that reason deserved a new method name. I am aware there is some code duplication between trace.Begin and Auditf, but I felt like keeping Auditf separate from the trace constructor allows us some flexibility on how to log additional information at INFO level while keeping the standard trace.Begin message at DEBUG level.

Reply:
I think the PR I posted still allows for flexibility on logging. It's functionally equivalent — just with less duplicated code.